### PR TITLE
Added Spinpanel and connector files to the certified connectors folder to prepare for certification

### PR DESCRIPTION
**Pull request by VincentMejan**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by ptalrejagithub** [comment]
Thank you for your submission. The files will be reviewed, and we will get back to you within two business days. All communications will be done in GitHub in the open-source phase.

File: certified-connectors/Spinpanel/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2047 @@
...
+                              "type": "string",
+                              "description": "servicePlanId"
+                            },
+                            "servicePlanName": {
+                              "type": "string",
+                              "description": "servicePlanName"
+                            },
+                            "provisioningStatus": {
+                              "type": "string",
+                              "description": "provisioningStatus"
```
Comment (sriyen-msft) [inline certified-connectors/Spinpanel/apiDefinition.swagger.json:794]: @VincentMejan All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Spinpanel/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2047 @@
...
+        ],
+        "description": "Create azure AD group."
+      }
+    },
+    "/directory/v1/organizations/{organizationId}/usergroups/{userGroupId}": {
+      "delete": {
+        "responses": {
+          "204": {
+            "description": "Spinpanel usergroup removed",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Spinpanel/apiDefinition.swagger.json:1536]: @VincentMejan Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: certified-connectors/Spinpanel/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2047 @@
...
+            "x-ms-summary": "Microsoft tenant id of the customer.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "quantityOnly",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "description": "",
```
Comment (sriyen-msft) [inline certified-connectors/Spinpanel/apiDefinition.swagger.json:1986]: @VincentMejan Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Spinpanel/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "1d2b61d3-a040-4c69-b597-b20aad4dd0b0",
```
Comment (sriyen-msft) [inline certified-connectors/Spinpanel/apiProperties.json:8]: @VincentMejan Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Spinpanel/apiDefinition.swagger.json (hunk 1 above)
Comment (VincentMejan) [reply certified-connectors/Spinpanel/apiDefinition.swagger.json]: Updated descriptions

File: certified-connectors/Spinpanel/apiDefinition.swagger.json (hunk 3 above)
Comment (VincentMejan) [reply certified-connectors/Spinpanel/apiDefinition.swagger.json]: Removed empty value

File: certified-connectors/Spinpanel/apiProperties.json (hunk 4 above)
Comment (VincentMejan) [reply certified-connectors/Spinpanel/apiProperties.json]: Replaced with [[DUMMY]]

File: certified-connectors/Spinpanel/apiDefinition.swagger.json (hunk 2 above)
Comment (VincentMejan) [reply certified-connectors/Spinpanel/apiDefinition.swagger.json]: Removed empty response schema where no actual content is returned by our APIs

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**